<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="FlipkartTestSuite" duration-ms="276" started-at="2021-04-23T18:41:12Z" finished-at="2021-04-23T18:41:13Z">
    <groups>
    </groups>
    <test name="Login" duration-ms="276" started-at="2021-04-23T18:41:12Z" finished-at="2021-04-23T18:41:13Z">
      <class name="com.web.locus.testcases.LocusTestRunner">
        <test-method status="PASS" signature="setUp(java.lang.String, org.testng.ITestContext)[pri:0, instance:com.web.locus.testcases.LocusTestRunner@175c2241]" name="setUp" is-config="true" duration-ms="3629" started-at="2021-04-23T18:41:09Z" finished-at="2021-04-23T18:41:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[src/main/resources/prod/prod.properties]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4a891c97]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verify()[pri:0, instance:com.web.locus.testcases.LocusTestRunner@175c2241]" name="verify" duration-ms="261" started-at="2021-04-23T18:41:12Z" finished-at="2021-04-23T18:41:13Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.util.BaseFunctions.click(BaseFunctions.java:48)
at com.web.pages.flipkart.FlipkartHomePage.login(FlipkartHomePage.java:27)
at com.web.locus.testcases.LocusTestRunner.verify(LocusTestRunner.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify -->
      </class> <!-- com.web.locus.testcases.LocusTestRunner -->
    </test> <!-- Login -->
  </suite> <!-- FlipkartTestSuite -->
</testng-results>
